presto-main/src/main/java/com/facebook/presto/sql/planner/plan/TableScanNode.java

```diff
@@ -40,6 +41,7 @@
 {
     private final TableHandle table;
     private final List<Symbol> outputSymbols;
```

Comment:
We don't need outputSymbols here. We can convert to List<Symbol> from List<VariableReferenceExpression> very easily.

Reply:
I tried that first. It means a lot of unnecessary code change. I think adding variable separate from symbol then remove symbol is cleaner.